tsconfig.json

```diff
@@ -16,5 +16,8 @@
     "strict": true,
     "target": "es2020"
   },
-  "include": ["src", "tests"]
+  "include": ["src", "tests"],
+  "ts-node": {
+    "esm": true
+  }
```

Comment:
`ts-node` was fighting me with the publish script. If anyone has more familiarity there, open to feedback.